tests/tests_unit/test_utils/test_time.py

```diff
@@ -35,6 +37,30 @@
     import pandas
 
 
+class TestDatetimeToMsIsoTimestamp:
+    @pytest.mark.skipif(platform.system() == "Windows", reason="Overriding timezone is too much hassle on Windows")
```

Comment:
If this `skipif` is used for all tests in this class, I think you can change it to a class decorator instead to avoid repetition!

Reply:
Some tests do not use this, and those can be run on Windows (with fingers crossed...)